cfgov/v1/image_formats.py

```diff
@@ -0,0 +1,11 @@
+from wagtail.images.formats import Format, register_image_format
+
+
+register_image_format(
```

Comment:
I have a small concern about adding the ability to add images in this format inside rich text fields. Most notably, it requires that alt text be added for each inserted image, which is a minor accessibility concern if the image is decorative (i.e., a screen reader reading text about the image wouldn't add value for the user). Aside from that, we have more control over the markup output when using our custom ContentImage block.

All that said, I know it can be annoying to have to break up long stretches of text with separate image blocks (when editing in the Wagtail interface), so it's probably not the end of the world. Just try to make the alt text as useful or unobtrusive as possible.

Reply:
I agree with you and if we continue doing more "story" like content on cf.gov we'll want to explore more standardized methods of embedding images/diagrams/videos.

Reply:
I'm late on this, but I just wanted to note that, as of Wagtail 2.13 (which we'll be on soon), alt text is no longer required for rich text images. We'll be able to insert decorative images with a correct, blank alt attribute once the update to 2.13 is done.